assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -391,6 +394,25 @@ def get_model_signature(self):
             ),
         )
 
+    def _robust_load_config(self, config) -> transformers.PretrainedConfig:
+        """Check and save if config has missing indices.
+
+        :param config: Config Object from transformers
+        :type config: PretrainedConfig
+        :return: Config Object
+        :rtype: PretrainedConfig
+        """
+        ids = list(config.id2label.keys())
+        ids.sort()
+        if max(ids) != len(ids)-1:
+            id2label = {}
+            for idx, id in enumerate(ids):
+                id2label[idx] = config.id2label[id]
+            config.id2label = id2label
+            self.config = config
+            logger.warning("config loaded with modified id2label as there are some missing keys.")
```

Comment:
Would be good to call out, which ones were missing or added by us here/

Reply:
done, thanks